kernel/build/config-amd64

```diff
@@ -2180,7 +2180,8 @@ CONFIG_BLK_DEV_NBD=m
 CONFIG_XEN_BLKDEV_FRONTEND=y
 CONFIG_VIRTIO_BLK=y
 CONFIG_BLK_DEV_RBD=y
-# CONFIG_BLK_DEV_UBLK is not set
+CONFIG_BLK_DEV_UBLK=m
+CONFIG_BLKDEV_UBLK_LEGACY_OPCODES=y
```

Comment:
this sounds like bad from security perspective

Reply:
I was going to drop it